chapters/data-analysis.tex

```diff
@@ -329,61 +341,61 @@ \subsection{Documenting variable construction}
 \section{Writing data analysis code}
 
 % Intro: this section focuses on data analysis CODE
-When data is cleaned and indicators constructed, you are ready to generate analytical outputs.
+After data is cleaned and indicators are constructed, you are ready to generate analytical outputs.
 \index{data analysis}
-There are many existing resources for data analysis, such as
-\textit{R for Data Science};\sidenote{\url{https://r4ds.had.co.nz}}
-\textit{A Practical Introduction to Stata};\sidenote{\url{https://scholar.harvard.edu/files/mcgovern/files/practical\_introduction\_to\_stata.pdf}}
-\textit{Mostly Harmless Econometrics};\sidenote{\url{https://www.researchgate.net/publication/51992844\_Mostly\_Harmless\_Econometrics\_An\_Empiricist's\_Companion}}
+There are many existing resources for data analysis and statistical methods, such as
+\textit{R for Data Science};\sidenote{\cite{hadley2017R} Available at \url{https://r4ds.had.co.nz}}
+\textit{A Practical Introduction to Stata};\sidenote{\cite{RePEc:gdm:wpaper:9412}}
+\textit{Mostly Harmless Econometrics};\sidenote{\cite{angrist2008mostly}}
```

Comment:
I don't think cites need to be sidetones, check how this compiles

Reply:
done in da94424